vbfml/scripts/train.py

```diff
@@ -44,39 +49,25 @@ def cli(ctx, tag):
     "--learning-rate", default=1e-3, required=False, help="Learning rate for training."
 )
 @click.option("--dropout", default=0.5, required=False, help="Dropout rate.")
-def setup(ctx, learning_rate: float, dropout: float):
+@click.option(
+    "--input-dir",
+    default="./output/2021-10-29_vbfhinv_26Oct21_nanov8",
+    required=False,
+    help="Input directory containing the ROOT files for training and validation.",
+)
+@click.option(
```

Comment:
Instead of having a predefined set of choices here, why not just feed in a configuration file?
```
train.py --model-config /path/to/config.yml
```

Then, we can have entirely separate config files for each type of architecture. In the current proposal, the config file has settings for both convolutional an ddense models which makes no sense. If we split, we can have one common key for all architectures that tells the parser what type of model to build, e.g.:
```
architecture: conv
```

this will also give us flexibility because we can now easily add different types of new architectures

Reply:
Thanks, I split the configuration files for each model in 88caa8d